qutip/core/metrics.py

```diff
@@ -440,6 +440,9 @@ def dnorm(A, B=None, solver="CVXOPT", verbose=False, force_solve=False,
 
     The diamond norm SDP is solved by using `CVXPY <https://www.cvxpy.org/>`_.
 
+    If B is provided and both A and B are unitaries, a special optimised
+    case of the diamond norm is used.
```

Comment:
Please add the reference to the papers, etc with the method here too.

Reply:
Sounds good!